website/docs/addon_aftereffects_artist.md

```diff
@@ -24,6 +24,11 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 
 To install the extension, download, install [Anastasyi's Extension Manager](https://install.anastasiy.com/). Open Anastasyi's Extension Manager and select AfterEffects in menu. Then go to `{path to pype}hosts/aftereffects/api/extension.zxp`. 
 
+(Current location will be most likely in artist `AppData`, on Windows it would be something like `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\openpype_X.X.X\openpype\hosts\aftereffects\api`.
```

Comment:
Tbh, I'm not sure if we should use `core` instead of OpenPype..
```suggestion
(Current location will be most likely in artist `AppData`, on Windows it would be something like `c:\Users\YOUR_USER\AppData\Local\Ynput\AYON\addons\core_X.X.X\ayon_core\hosts\aftereffects\api`.
```